common/bin/oo-diagnostics

```diff
@@ -1207,14 +1207,16 @@ class OODiag
   def test_broker_certificate
     skip_test unless @is_broker
 
+    broker_host_env = File.read("/etc/openshift/env/OPENSHIFT_BROKER_HOST")
+
     # Retrieve the SSL cert from Apache
     response = `curl -k -s -v https://localhost/ -o /dev/null 2>&1`
     if $? != 0
       do_fail <<-CANNOTCONNECT
-        An error arose while trying to connect to the SSL reverse proxy.  curl returned with
+        Attempted to connect to #{broker_host_env} but failed. curl returned with
         #{response.strip.empty? ? "no output." : "the following output:\n" + response}
 
-        Please check whether the httpd service is running.
+        Please check whether the httpd service in #{broker_host_env} is running
```

Comment:
Looks like you made the change in the output for localhost, but meant to make the change in the output for `#{broker_host_env}`